packages/api/src/resolvers/mutations/filters/createFilter.ts

```diff
@@ -1,20 +1,23 @@
 import { log } from '@unchainedshop/logger';
 import { Root, Context } from '@unchainedshop/types/api.js';
-import { Filter } from '@unchainedshop/types/filters.js';
+import { Filter, FilterInputText } from '@unchainedshop/types/filters.js';
 
 export default async function createFilter(
   root: Root,
-  { filter }: { filter: Filter & { title: string } },
+  { filter, texts }: { filter: Filter; texts: Array<FilterInputText> },
```

Comment:
Naming is not consistent, here it's Array<FilterInputTexts> and for Assortments it's AssortmentText[].

Stick to one way, for ex. Array<AssortmentInputTexts> or FilterText[]